.github/workflows/build.yaml

```diff
@@ -11,18 +11,18 @@ defaults:
     shell: bash
 
 env:
-  ARM_TOOLS_URL: https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu-rm/9-2020q2/gcc-arm-none-eabi-9-2020-q2-update-x86_64-linux.tar.bz2
-  ARM_TOOLS_DIR: gcc-arm-none-eabi-9-2020-q2-update
+  ARM_TOOLS_URL: https://developer.arm.com/-/media/Files/downloads/gnu-rm/10-2020q4/gcc-arm-none-eabi-10-2020-q4-major-x86_64-linux.tar.bz2
```

Comment:
Took the liberty of upgrading to the latest toolchains. upgrade instructions in readme but for most things using the older version _should_ be fine